index.html

```diff
@@ -78,15 +78,15 @@ <h1>Have a skin concern?</h1>
 			<div class="similar">
 				<!-- Flexbox to divide the three recommended articles easier-->
 				<div class="flex-container">
-					<div class="flex-child"></div>
-					<div class="flex-child"></div>
+					<div class="flex-child"><img src="images/fungal_acne.jpg"></div>

```

Comment:
Added the 2 newly written articles in the "recommended articles" boxes with functioning links on the Index page.